Handling stop words and stemming in different languages (#26)
OnTheGoSystems/wpml-elasticpress: README.md

* As posts are indexed in the default language, syncronization can mess with features like stopwords and stemming, resulting in wrongly indexed data.

The purpose of this plugin is providing proper indexing for the content based on its own language.

To do so, this glue plugin appends two additional fields to indexed documents:
* `post_lang` stores all the languages where a post might appear as a search result. This includes the post own language, but also some additional languages in case your post type is set to display as translated when some translations are missing. Posts on non-translatable types store here the default language so they can appear on frontend search results.
* `post_unique_lang` stored the actual, single language of the each post. Posts on non-translatable types store here the default language.

## Additional CLI command

This plugin provides its own `wpml_elasticpress` WP CLI command that extends the `elasticpress` one. When using `wpml_elasticpress`, you can pass some extra attributes:

> * `wp wpml_elasticpress sync` will index all your posts in their default language. The process is run on batches, and languages are indexed one after another.

Reviewer: ```suggestion
* `wp wpml_elasticpress sync` will index all your posts in their default language. The process runs in batches, and languages are indexed one after another.
```

Author: +1

## Changelog

### 2.0.0

> - **Breakign change**: introduce a different index per language.

Author: +1
